internal/pkg/cmd/prerunner.go

```diff
@@ -233,6 +232,9 @@ func IsFlagRequired(flag *pflag.Flag) bool {
 // Authenticated provides PreRun operations for commands that require a logged-in Confluent Cloud user.
 func (r *PreRun) Authenticated(command *AuthenticatedCLICommand) func(*cobra.Command, []string) error {
 	return func(cmd *cobra.Command, args []string) error {
+		if err := r.Config.DecryptContextStates(); err != nil {
```

Comment:
added this step for other prerunner operations except anonymous ones.

Reply:
Does this have to go before `Anonymous()`? It's a bit easier to reason about things when `Anonymous()` is called first.